eng/testing/performance/performance-setup.ps1

```diff
@@ -172,6 +172,7 @@ if ($iOSMono) {
         Copy-Item -path "$SourceDirectory\iosHelloWorld\nollvm" $PayloadDirectory\iosHelloWorld\nollvm -Recurse
         Copy-Item -path "$SourceDirectory\MauiiOSDefaultIPA" $PayloadDirectory\MauiiOSDefaultIPA -Recurse
         Copy-Item -path "$SourceDirectory\MauiMacCatalystDefault\MauiMacCatalystDefault.app" $PayloadDirectory\MauiMacCatalystDefault -Recurse
+        Copy-Item -path "$SourceDirectory\MauiiOSPodcastIPA" $PayloadDirectory\MauiiOSPodcastIPA -Recurse
```

Comment:
This section of performance-setup seems awfully test case specific for a script with such a general name. Probably naively optimistic of me, but are there any ways we could move move test case specific knowledge closer to the test cases?

Reply:
I think it's probably possible to take care of a lot of this with the yml templates, so we have a template that just takes a list of apps, and expands to the various stuff for them. 

I was reluctant to try to do that given deadlines and the difficulty in debugging yml templates though. It would be a good thing to tackle in the future.

Reply:
I added this as an area of improvement: https://github.com/dotnet/performance/issues/2113. This is 100% something we should at least look into.

Reply:
Yeah, in particular we are impacted by the fact that we only want to build the apps once, but use them in various test cases. That goes against some of the assumptions around how our scenario support was built. We don't need to design the feature in this comment thread, but I think that trying to bring the concept of pre-building dependencies of scenarios into our current python scenario execution code is where we should focus our efforts.